bank/postcss.config.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    plugins: {
+        tailwindcss: {},
+        autoprefixer: {},
+    },
+};
```

Comment:
Storybook throws a warning for postcss addon, I mean I might be a bug, but does it show only by me?

Also found an [issiue](https://github.com/storybookjs/addon-postcss/issues/45) with the same problem, but does not tell how to solve it since it is pretty new.

```
bank-web-1  | @storybook/cli v7.0.2
bank-web-1  | 
bank-web-1  | Attention: We've detected that you're using the following addons in versions which are known to be incompatible with Storybook 7:
bank-web-1  | 
bank-web-1  | - @storybook/addon-postcss@2.0.0
bank-web-1  | 
bank-web-1  | Please be aware they might not work in Storybook 7. Reach out to their maintainers for updates and check the following Github issue for more information:
bank-web-1  | https://github.com/storybookjs/storybook/issues/20529

```

Reply:
warning in JS 😂😂
But on the more serious note, if the UI works for you I'd ignore the warning, I've had no problems but couldn't check dockerized version.